src/injected/visualization/svg-drawer.ts

```diff
@@ -86,23 +85,22 @@ export class SVGDrawer extends BaseDrawer {
             oldStateElement == null ||
             newStateElement.target[newStateElement.target.length - 1] !==
                 oldStateElement.selector ||
-            newStateElement.tabOrder !== oldStateElement.tabOrder ||
+            newStateElement.propertyBag?.tabOrder !== oldStateElement.tabOrder ||
             isLastElementInSvg
         );
     }
 
     private getNewTabbedElement(
         oldStateElement: TabbedItem,
-        newStateElement: TabbedElementData,
-        pos: number,
```

Comment:
Is removing this intentional?

Reply:
Yes, it doesn't appear to be used anywhere so I figured it might as well be removed, unless there's a reason to keep it I'm not aware of?